styles.css

```diff
@@ -0,0 +1,132 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;1,500&family=Share+Tech&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+body {
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  background-color: #f9fafb;
+}
+
+
+.side {
+  position: relative;
+  background-image: url(./images/background.jpg);
+  background-size: cover;
+  width: 40vw;
+  height: 130vh;
+}
+
+.logo {
+  background-color: #00000066;
+  color: #fff;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: 20vh;
+}
+
+.logo img {
+  width: 5vw;
+}
+
+.logo-text {
+  font-size: 4rem;
+}
+
+.credit {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.photo-credit {
+  color: #fff;
+  position: absolute;
+  bottom: 1rem;
+  font-size: 1rem;
+}
+
+.photo-credit a {
+  color: #fff;
+}
+
+
```

Comment:
remove this extra line, use `prettier` plugin in vs code to get read of bad formatting